### PR TITLE
Fix test_extreme_deduplication flakiness

### PR DESCRIPTION
**Pull request by azat**
This is part of **"Why my PR is not merged?"** patch set series.

Since #50107 there one more tunable that can increase the cleanup period
- max_cleanup_delay_period.

Plus, increase the delay for the cleanup thread, otherwise it is too fragile.

CI: https://s3.amazonaws.com/clickhouse-test-reports/51514/b03d813daae4c4ab047380f49f5a64be3336fd24/integration_tests__asan__[4_6].html

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by azat** [comment]
>Integration tests flaky check (asan) — No flaky tests: error: 0, passed: 1, fail: 0, skipped: 1, flaky: 0, broken: 0, not_fail: 0

Looks OK

**Comment by azat** [comment]
>Stateless tests (aarch64) — fail: 1, passed: 5399, skipped: 24

- `02160_special_functions`

```
2023-07-13 07:41:45 --- /usr/share/clickhouse-test/queries/0_stateless/02160_special_functions.reference	2023-07-13 07:32:10.316533192 -0700
2023-07-13 07:41:45 +++ /tmp/clickhouse-test/0_stateless/02160_special_functions.stdout	2023-07-13 07:41:45.221717785 -0700
2023-07-13 07:41:45 @@ -41,6 +41,6 @@
2023-07-13 07:41:45  701
2023-07-13 07:41:45  800
2023-07-13 07:41:45  60200201
2023-07-13 07:41:45 -60
2023-07-13 07:41:45 -10
2023-07-13 07:41:45 +-3599999940
2023-07-13 07:41:45 +-3590
2023-07-13 07:41:45  1
2023-07-13 07:41:45 
```

https://github.com/ClickHouse/ClickHouse/issues/42653

File: tests/integration/test_extreme_deduplication/test.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def test_deduplication_window_in_seconds(started_cluster):
     assert TSV(node.query("SELECT count() FROM simple")) == TSV("2\n")
 
     # wait clean thread
-    time.sleep(2)
+    time.sleep(5)
```
Comment (alexey-milovidov) [inline tests/integration/test_extreme_deduplication/test.py:53]: Do we need to add a loop?, because synchronization with sleep looks not sufficient.

**Comment by alexey-milovidov** [comment]
Let's merge the changes from https://github.com/ClickHouse/ClickHouse/pull/51992/files

**Pull request merged**